python/paddle/jit/sot/opcode_translator/executor/instr_flag.py

```diff
@@ -34,3 +35,21 @@ class MAKE_FUNCTION_FLAG:
 
 class CALL_FUNCTION_EX_FLAG:
     CFE_HAS_KWARGS = 0x01
+
+
+class Intrinsics_UnaryFunctions(Enum):
```

Comment:
```suggestion
class IntrinsicsUnaryFunctions(Enum):
```

这里按照 Python 代码风格

另外加一下注释，这里的是从 cpython 哪里 copy 来的，以便未来版本适配时更新

Reply:
下一个 PR 加一下 binary 的，但是全部 Fallback 吧